pkg/cloud/aws/actuators/cluster/actuator.go

```diff
@@ -154,6 +210,7 @@ func (a *Actuator) Delete(cluster *clusterv1.Cluster) error {
 	if err := ec2svc.DeleteNetwork(); err != nil {
 		a.log.Error(err, "Error deleting cluster", "cluster-name", cluster.Name, "cluster-namespace", cluster.Namespace)
 		return &controllerError.RequeueAfterError{
+			// TODO: use a Duration expression for this
 			RequeueAfter: 5 * 1000 * 1000 * 1000,
```

Comment:
Can we actually change this as part of this PR?

Reply:
Sure, that's 5 seconds, right?

Reply:
Correct, had to make sure though :) https://play.golang.org/p/LFjZSdONubV